base/reduce.jl

```diff
@@ -270,6 +270,19 @@ should be evaluated as `(1-2)-3` or `1-(2-3)`. Use [`foldl`](@ref) or
 Some operations accumulate error, and parallelism will also be easier if the reduction can
 be executed in groups. Future versions of Julia might change the algorithm. Note that the
 elements are not reordered if you use an ordered collection.
+
+# Examples
+
+```jldoctest
+julia> reduce(+, 1, [2; 3; 4])
```

Comment:
the "unspecified"-ness about `v0` means it's probably best practice to use 0 for addition